src/components/page/bookList.jsx

```diff
@@ -53,31 +21,18 @@ const BookList = () => {
 
   const handleSubmit = (e) => {
     e.preventDefault();
-    const bookCollection = [...books];
     const newBook = {
       id: books.length,
```

Comment:
Oops, using `books.length`, I'm afraid we won't have a unique `id`, try to console.log id, delete a book then add a new book.

You might want to use `Math.random` or automatically generate ids using [npm uuid](https://www.npmjs.com/package/uuidv4)